lib/generators/hyrax/work_resource/templates/work.html.erb_spec.rb.erb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
```

Comment:
Should we also add the `Generated via` comment that is in the other generated files here?

Reply:
I think so.

Reply:
:+1:

Reply:
:+1:

Reply:
```suggestion
# Generated via
#  `rails generate hyrax:work <%= class_name %>`
```